fixit/common/config.py

```diff
@@ -70,22 +80,25 @@
 def get_validated_settings(
     file_content: Dict[str, Any], current_dir: Path
 ) -> Dict[str, Any]:
+    try:
+        # __package__ should never be none (config.py should not be run directly)
+        # But use .get() to make pyre happy
+        pkg = globals().get("__package__")
+        assert pkg, "No package was found, config types not validated."
+        config = pkg_resources.read_text(pkg, "config.schema.json")
+        # Validates the types and presence of the keys
+        schema = json.loads(config)
+        validate(instance=file_content, schema=schema)
+    except (AssertionError, FileNotFoundError):
```

Comment:
I don't think this is the best thing to do, but I could not figure out how to get `tox -e docs` happy with the newly added file. If anyone understands how `tox` is setup better there might be a way to avoid the missing file error.

Reply:
With my suggested changes, those try except can be removed.
Verified on my dev server with `tox -e docs`.